### DISCUSSION TITLE
how to increase font size ?

### DISCUSSION THREAD

**Discussion by pchalasani** [squidfunk/mkdocs-material#6170] [Q&A]
This is a very basic question I'm struggling with. I created docs for my site here:
https://langroid.github.io/langroid/

The font size looks smaller than many other sites using mkdocs material, such as:
https://docs.guardrailsai.com/

I think the larger font size makes docs look more approachable. In any case, I have been comparing the settings in the two repos and cannot figure out how to make the fonts larger.

Any help/pointers will be much appreciated!

**Comment by kamilkrzyskow** [comment]
What exactly do you mean by "font size looks smaller"?
I compared 2 paragraphs from both websites:
> "Effectively leveraging LLMs at scale requires a principled programming framework. In particular, there is often a need to maintain multiple LLM conversations, each instructed in different ways, and "responsible" for different aspects of a task." 

> Guardrails AI is a fully open source library that assures interactions with Large Language Models (LLMs). It offers

and both loo same to me, and the browser detects the _computed_ size of 16px with the Roboto font 🤔 
None of the websites make modifications to the font size and both are built with the Material theme 9.4
Your website changes the size of the image, so maybe there is a bigger contrast between the enlarged header and the font
https://github.com/langroid/langroid/blob/main/docs/stylesheets/extra.css

**Reply by alexvoss** [reply]
They look the same to me, too. Also opened the developer tools and found the same computed sizes. I am not entirely sure if there is a single control that you could use to modify all font sizes, though a lot of them are derived from the `font-size` setting in `.md-typeset`. That changes the `typeset` content only, though, and I don't think it changes admonitions as it probably should.

**Comment by pchalasani** [comment]
You're both right, yes they are the same. I thought I had checked it but it turned out I was using 90% maginification in the page with my docs. But in general, good to know how to adjust fonts and sizes of the different elements.
